### DISCUSSION TITLE
Error in setting the developer environment

### DISCUSSION THREAD

**Discussion by patel85110** [JdeRobot/RoboticsAcademy#1996] [Q&A]
Hello sir,

I have followed all the steps from [InstructionsForDevelopers.md](https://github.com/JdeRobot/RoboticsAcademy/blob/master/docs/InstructionsForDevelopers.md) but still facing this error while running server.

![Screenshot from 2023-04-02 13-24-04](https://user-images.githubusercontent.com/129650294/229340153-6e1b9429-8b49-4e24-836f-a21866427534.png)

any solution for this ??

**Comment by pariaspe** [comment]
Hi @patel85110 , 
Make sure to install all needed dependencies at [requirements.txt](https://github.com/JdeRobot/RoboticsAcademy/blob/master/scripts/requirements.txt).
